uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -210,5 +210,4 @@ our $HRRT_ALREADY_INSERTED          = 285; # if HRRT archive already exists in D
 our $HEADER_INSERT_FAILURE          = 286; # if could not insert matlab info into the
                                            # MINC header
 our $HRRT_ARCHIVE_INSERTION_FAILURE = 287; # if HRRT archive insertion into the
-                                           # database has failed
-our $UNKNOW_PROTOCOL                = 288; # if HRRT protocol is not found
```

Comment:
$UNKNOW[N]_PROTOCOL already exists.

Reply:
I am guessing that `UNKNOW_PROTOCOL ` is not used anywhere else. Just to be sure.

Reply:
Yes, I checked.